operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md

```diff
@@ -0,0 +1,38 @@
+# Change Air-Cooled Node BMC Credentials
+
+This procedure will use the System Configuration Service (SCSD) to change all air-cooled Node BMCs in the system to the same global credential. 
+
+**Limitation:** All air-cooled and liquid cooled BMCs should share the same global credentials. The air-cooled Slingshot Switch Controllers (Router BMCs) must have the same credentials as the liquid-cooled Slingshot switch controllers.
```

Comment:
This should be a separate header and the text requires a couple of consistency edits. See below.

### Limitations 

All air-cooled and liquid-cooled BMCs share the same global credentials. The air-cooled Slingshot switch controllers (Router BMCs) must have the same credentials as the liquid-cooled Slingshot switch controllers.

Reply:
Should be resolved fixed now.